libyelp/yelp-uri.c

```diff
@@ -856,20 +856,25 @@ resolve_ghelp_uri (YelpUri *uri)
 static void
 resolve_help_uri (YelpUri *uri)
 {
-    /* help:document[/page][?query][#frag]
+    /* help:[/*]document[/page][?query][#frag]
```

Comment:
So what's the alternative here? Should I avoid documenting the actual change just because `-Werror=comment` don't like this? Or should I switch to c++-like comments? I can pass `-Wno-error=comment`, too.